<p>For readability reasons, the starting token <code>#</code> should be followed by a whitespace.</p>

<h2>Noncompliant Code Example</h2>
<pre>
#My comments...
</pre>

<h2>Compliant Solution</h2>
<pre>
# My comments...
</pre>
