<p>
    Having a feature that does not define any scenario is useless. It can either be safely removed or scenarios should
    be added.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Feature: My feature...
</pre>

<h2>Compliant Solution</h2>
<pre>
Feature: My feature...
  Scenario: My scenario #1
    ...
</pre>
